epochStart/shardchain/trigger.go

```diff
@@ -260,6 +263,7 @@ func NewEpochStartTrigger(args *ArgsShardEpochStartTrigger) (*trigger, error) {
 		appStatusHandler:              args.AppStatusHandler,
 		roundHandler:                  args.RoundHandler,
 		enableEpochsHandler:           args.EnableEpochsHandler,
+		extraDelayForRequestBlockInfo: args.ExtraDelayForRequestBlockInfo,
```

Comment:
I think we should add a warning in the logs if we provide a different value of 3 seconds for this (as seen in the chain simulator)
Reason: If we change this in the future, at least we should have a warning that the epoch change trigger might not behave normally.

Reply:
done